tools/representations.py

```diff
@@ -300,6 +329,12 @@ def has_subdivision_name(self, subdivision_name):
     def has_country_code(self, country_code):
         return self.country_code == country_code
 
+    def has_feature(self, feature):
+        return feature in self.features
```

Comment:
To handle the `None` type, the code would look like this.
```suggestion
        return feature in self.features if self.features is not None else False
```